compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/Configuration.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.uikit
+
+class Configuration {
```

Comment:
Configuration of what? Also, public class requires docs

Reply:
Added doc comment.

I don't know how it is better to call.
It is a top level configuration of ComposeUIViewController() function.
Maybe call it "ComposeUIViewControllerConfiguration" ?